operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/reconciler/PrimaryUpdateAndCacheUtils.java

```diff
@@ -25,6 +27,8 @@
 public class PrimaryUpdateAndCacheUtils {
 
   public static final int DEFAULT_MAX_RETRY = 10;
+  public static final int RESOURCE_CACHE_POLL_TIMEOUT = 10000;
```

Comment:
@shawkins do you know some real life data, if on how long it would take on larger cluster to an informer to receive the event? On small one is almost instant (if watch works), not sure how the cluster size influences this.